socs/agents/acu/agent.py

```diff
@@ -191,7 +201,18 @@ def __init__(self, agent, acu_config='guess',
             if v is not None:
                 self.datasets[k] = dict(_dsets['datasets'])[v]
 
-        self.monitor_fields = status_keys.status_fields[self.platform_type]['status_fields']
+        # Create a map from each status key (read through the
+        # self.datasets) to the output block and field name.
+        self.status_field_map = {}
+        for group, group_fields in \
+                status_keys.status_fields[self.platform_type]['status_fields'].items():
+            for block_name, block_group, _, _ in MONITOR_STRUCTURE:
+                if block_group == group:
+                    break
+            else:
+                raise ValueError(f"status_key block '{group}' not configured for monitoring.")
```

Comment:
This looks like it'll crash the agent on start up. Is that the desired behavior? And if a `status_key` isn't configured, that means we just need to add a line for it in `MONITOR_STRUCTURE`?